tests/phpunit/elementor/data/base/test-controller.php

```diff
@@ -188,5 +188,26 @@ public function test_get_items_recursive_simulated() {
 		}
 	}
 
-	// TODO: test_get_permission_callback.
+	public function test_get_permission_callback() {
+		// Set admin.
+		wp_set_current_user( $this->factory()->create_and_get_administrator_user()->ID );
+
+		$controller = new ControllerSimple();
+		$controller->bypass_original_permission( false );
+
+		$methods = explode( ', ', \WP_REST_Server::ALLMETHODS );
+
+		foreach( $methods  as $method ) {
+			$request = new \WP_REST_Request( $method );
+			$this->assertEquals( $controller->get_permission_callback( $request ), true );
+		}
+
+		// Set subscriber.
+		wp_set_current_user( $this->factory()->get_subscriber_user()->ID );
```

Comment:
Add a test for `editor` rule.

Reply:
Handled: https://github.com/elementor/elementor/pull/11754/commits/a97a12c33811bbc4881a0b731c1b14c650a6b565